.github/workflows/main.yml

```diff
@@ -55,6 +58,18 @@ jobs:
           python manage.py restore_postgresql_database
           # run indexing indexing
           python manage.py get_index_data_from_database -b False
+          # set up the seond database
+          rm app_data/omero_db_searchengine.zip
+          rm app_data/omero.pgdump
+          wget https://github.com/khaledk2/ice-archh-64/releases/download/new_re_db/omero_train.zip -P app_data
```

Comment:
Long-term I assume this is an issue. I wonder to what extent we couldn't try omero-test-infra for the github tests and then do something larger on our own systems.